### PR TITLE
bt-502: Mobile - Add verification message after onboarding steps for both roles

### LINKED ISSUE #502: Mobile - Add verification message after onboarding steps for both roles
After a user completes his onboarding steps the “Waiting for approval” label appears near the avatar on the top, like in Djinni
In this state, until he is verified, he won't be able to reach other pages than the Preview of his profile.

Image

### PR DESCRIPTION
**Pull request by illia-dobro**
label

### CODE REVIEW

**Comment by katerynakostikova** [comment]
Let's make some screen header like in candidates screen and put that tag there, it should also be like warning (some yellow colors)

File: mobile/src/bundles/common/components/verification-message/styles.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { Color } from '~/bundles/common/enums/enums';
+import { StyleSheet } from '~/bundles/common/styles/styles';
+
+const styles = StyleSheet.create({
+    text: {
+        position: 'absolute',
+        top: 10,
+        right: 10,
```
Comment (katerynakostikova) [inline mobile/src/bundles/common/components/verification-message/styles.ts]: let's avoid absolute positioning, we can just align items center in header

File: mobile/src/bundles/common/components/verification-message/styles.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        borderRadius: 999,
```
Comment (katerynakostikova) [inline mobile/src/bundles/common/components/verification-message/styles.ts]: why 999?

File: mobile/src/bundles/common/components/verification-message/styles.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        backgroundColor: Color.WARNING,
+        color: '#FFFFFF',
+        shadowColor: '#000000',
+        shadowOffset: {
+            width: 0,
+            height: 2,
+        },
+        shadowOpacity: 0.25,
+        shadowRadius: 3.84,
```
Comment (katerynakostikova) [inline mobile/src/bundles/common/components/verification-message/styles.ts]: it's ios settings we don't need it

File: mobile/src/bundles/common/components/verification-message/verification-message.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    }
+
+    return (
+        <Text
+            category={TextCategory.CAPTION}
+            style={[styles.text, globalStyles.p5, globalStyles.ph10]}
+        >
+            Waiting for approval
+        </Text>
+    );
```
Comment (katerynakostikova) [inline mobile/src/bundles/common/components/verification-message/verification-message.tsx]: let's add this check directly in component where we use it

File: mobile/src/bundles/employer/screens/employer-profile/employer-profile.tsx [hunk 5]
```diff
@@ -1,9 +1,17 @@
...
+import {
+    Text,
+    VerificationMessage,
+} from '~/bundles/common/components/components';
 
 const EmployerProfile: React.FC = () => {
-    return <Text>Employer Profile screen</Text>;
+    return (
+        <>
+            <VerificationMessage isApproved={false} />
```
Comment (katerynakostikova) [inline mobile/src/bundles/employer/screens/employer-profile/employer-profile.tsx]: can we get this data from the store? We already store user data there

File: mobile/src/bundles/talent/store/slice.ts [hunk 6]
```diff
@@ -14,11 +14,13 @@ import {
 type State = {
     dataStatus: ValueOf<typeof DataStatus>;
     onboardingData: UserDetailsGeneralResponseDto | null;
+    isApproved: boolean; //todo when real data appear
 };
 
 const initialState: State = {
     dataStatus: DataStatus.IDLE,
     onboardingData: null,
+    isApproved: false, //todo when real data appear
```
Comment (katerynakostikova) [inline mobile/src/bundles/talent/store/slice.ts]: we still don't have it?

**Review by katerynakostikova — CHANGES_REQUESTED** [review]

**Review by katerynakostikova — APPROVED** [review]

**Review by anogire — APPROVED** [review]

**Pull request merged**